pkg/resources/manifests.go

```diff
@@ -270,6 +270,33 @@ func GenerateDeploymentManifest(ctx context.Context, workspaceObj *kaitov1alpha1
 	labelselector := &v1.LabelSelector{
 		MatchLabels: selector,
 	}
+	initContaiers := []corev1.Container{}
+	envs := []corev1.EnvVar{}
+
+	if len(workspaceObj.Inference.Adapters) != 0 {
+		for _, adapter := range workspaceObj.Inference.Adapters {
+			// TODO: accept Volumes and url link to pull images
+			initContaier := corev1.Container{
+				Name:            adapter.Source.Name,
+				Image:           adapter.Source.Image,
+				Command:         []string{"/bin/sh", "-c", fmt.Sprintf("mkdir -p /data/%s && cp -r * /data/%s", adapter.Source.Name, adapter.Source.Name)},
```

Comment:
mkdir -p /data/%s && cp -r * /data/%s
This command copies everything in the image filesystem (indicated by *) to /data/%s, including files we don't want 


mkdir -p /data/%s && cp -r data/* /data/%s
Copies only the contents of data (in the image) to /data/%s